interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1649,6 +1649,15 @@ namespace cling {
     // Return a symbol's address, and whether it was jitted.
     std::string mangledName;
     utils::Analyze::maybeMangleDeclName(GD, mangledName);
+#if defined(LLVM_ON_WIN32)
+    // For some unknown reason, Clang 5.0 adds a special symbol ('\01') in front
+    // of the mangled names on Windows, making them impossible to find
+    // TODO: remove this piece of code and try again when updating Clang
+    if (mangledName.size() > 2 && mangledName[1] == '?' &&
+        mangledName.compare(1, 14, std::string("?__cling_Un1Qu"))) {
+      mangledName.erase(0, 1);
```

Comment:
```suggestion
           mangledName.erase(0, 1);
```

Reply:
@dpiparo why? the indent is two characters, starting from `if (mangledName.size() > 2 && ...`

Reply:
:( sorry, my bad.